bucket/winauth.json

```diff
@@ -6,6 +6,19 @@
     "url": "https://github.com/winauth/winauth/releases/download/3.6.2/WinAuth-3.6.2.zip",
     "hash": "sha512:8288a970946c149382c1d8919905fda5381d04f24a4ceed1ec4f3fc7fda0a7c8f5c1fc062b7b312dca55531eecf04327ca0743afbb82ff08acedf4c7c9c4ed1f",
     "bin": "WinAuth.exe",
+    "pre_install": [
+        "if(!(Test-Path(\"$persist_dir\\winauth.xml\"))) { New-Item \"$dir\\winauth.xml\" -type file | Out-Null ",
+        "    $env:LOCALAPPDATA, $env:APPDATA | ForEach-Object {",
```

Comment:
Why both appdatas?

> If your configuration file (winauth.xml, normally stored in your Windows roaming profile

https://winauth.github.io/winauth/

Reply:
Oh, I didn't figure out where the directory was, but now it looks like you should just need APPDATA